drivers/hmis_csv_importer/app/models/hmis_csv_importer/hmis_csv_cleanup/append_organization_id.rb

```diff
@@ -0,0 +1,50 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module HmisCsvImporter::HmisCsvCleanup
+  class AppendOrganizationId < Base
+    def cleanup!
+      organization_batch = []
+
+      organization_scope.find_each do |organization|
+        organization.OrganizationName = "#{organization.OrganizationName} (#{organization.OrganizationID})"
```

Comment:
should we handle empty names?
```suggestion
        organization.OrganizationName = "#{organization.OrganizationName.presence || 'Unnamed Organization'} (#{organization.OrganizationID})"
```

Reply:
`OrganizationName`, and `ProjectName`, are required fields in the CSV spec, so I don't think we need to handle that at this time.